wordpress-mobile/WordPress-iOS
# Migrates MenusServiceRemote + models + tests to WordPressKit

Fixes #7314 

This PR:
* Migrates the `RemoteMenu`, `RemoteMenuItem`, and `RemoteMenuLocation` models to the WPKit framework
* Migrates `MenusServiceRemote.h` and `MenusServiceRemote.m` to the WPKit framework
* Migrates the corresponding unit tests (`MenusServiceRemoteTests.m`) to the WPKit framework
* Fixes a stray documentation warning in `MediaServiceRemoteXMLRPC.m`

Ref #7301

**To test:**
1. Clean DerivedData.
2. Update CocoaPods repo with pod repo update.
3. Install the pods with pod install.
4. Run _all_ unit tests and make sure they are green.
5. Build & run the app.
6. On a .com site, verify you are able to view, create, update, and delete menus/menu items under the "Personalize" section of the site admin screen:

<img width="302" alt="napkin 42 07-07-17 4 20 48 pm" src="https://user-images.githubusercontent.com/154014/27977417-4622b74a-6330-11e7-887d-a2d0730dca2f.png">


Needs review, por favor @elibud. Thanks!! 😄
